impeller/aiks/canvas.cc

```diff
@@ -561,8 +561,13 @@ void Canvas::DrawTextFrame(const TextFrame& text_frame,
     color_text_contents->SetColorSourceContents(
         paint.color_source.GetContents(paint));
 
-    entity.SetContents(
-        paint.WithFilters(std::move(color_text_contents), false));
+    // TODO(bdero): This mask blur application is a hack. It will always wind up
```

Comment:
Do we have bugs tracking these?

Reply:
Whoops, thanks for the reminder. Filed https://github.com/flutter/flutter/issues/133297.